CribblyBackend/Properties/launchSettings.json

```diff
@@ -1,30 +1,11 @@
 ﻿{
-  "$schema": "http://json.schemastore.org/launchsettings.json",
-  "iisSettings": {
-    "windowsAuthentication": false,
-    "anonymousAuthentication": true,
-    "iisExpress": {
-      "applicationUrl": "http://localhost:44739",
-      "sslPort": 44326
-    }
-  },
-  "profiles": {
-    "IIS Express": {
-      "commandName": "IISExpress",
-      "launchBrowser": true,
-      "launchUrl": "weatherforecast",
-      "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
-      }
-    },
-    "cribbly_backend": {
-      "commandName": "Project",
-      "launchBrowser": true,
-      "launchUrl": "weatherforecast",
-      "applicationUrl": "https://localhost:5001;http://localhost:5000",
-      "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
-      }
+  "cribbly_backend": {
```

Comment:
This file caused me some errors. To solve, I had to wrap the `cribbly_backend` object in `"profiles"`:
```json
{
  "profiles": {
    "cribbly_backend": {
      ...
    }
  }
}
```

Reply:
I reverted the file back to how it was before this branch. It didn't change anything on my end, so hopefully that should resolve the issue